src/components/BookstoreContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+
+import Navbar from './Navbar';
+import BooksList from './BooksList';
+import Categories from './Categories';
+
+function BookstoreContainer() {
+  return (
+    <div className="bookstore-container">
+      <Router>
+        <Navbar />
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🔥 Well done!